docs/GettingStartedDocs/install_oe_sdk-Ubuntu_16.04.md

```diff
@@ -37,7 +37,7 @@ sudo ./sgx_linux_x64_driver.bin
 
 ### 3. Install the Intel and Open Enclave packages and dependencies
 ```bash
-sudo apt -y install clang-7 libssl-dev gdb libsgx-enclave-common libsgx-enclave-common-dev libprotobuf9v5 libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client open-enclave
+sudo apt -y install clang-7 libssl-dev gdb libsgx-enclave-common libsgx-enclave-common-dev libprotobuf9v5 libsgx-qe3-logic libsgx-pce-logic libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client open-enclave
```

Comment:
These packages should be pulled in as dependencies of the other libsgx packages (confirmed on Ubuntu 16.04). We should only include the minimal set we depend on in the instructions